scripts/release.py

```diff
@@ -0,0 +1,256 @@
+import argparse
+import logging
+import os
+import re
+import time
+from datetime import date
+
+import requests
+from OBAS_utils.release_utils import check_release, closeRelease, download_url
+
+logging.basicConfig(encoding="utf-8", level=logging.INFO)
+
+parser = argparse.ArgumentParser("release")
+parser.add_argument(
+    "branch_platform", help="The new version number of the release.", type=str
+)
+parser.add_argument(
+    "previous_version", help="The previous version number of the release.", type=str
+)
+parser.add_argument(
+    "new_version", help="The new version number of the release.", type=str
+)
+parser.add_argument(
+    "--dev", help="Flag to prevent pushing the release.", action="store_false"
+)
+args = parser.parse_args()
+
+previous_version = args.previous_version
+new_version = args.new_version
+branch_platform = args.branch_platform
+
+github_token = os.environ["GREN_GITHUB_TOKEN"]
+
+os.environ["DRONE_COMMIT_AUTHOR"] = "Filigran-Automation"
+os.environ["GIT_AUTHOR_NAME"] = "Filigran Automation"
+os.environ["GIT_AUTHOR_EMAIL"] = "automation@filigran.io"
+os.environ["GIT_COMMITTER_NAME"] = "Filigran Automation"
+os.environ["GIT_COMMITTER_EMAIL"] = "automation@filigran.io"
+
+# Platform
+logging.info("[platform] Starting the release")
+
+# Pom
+with open(
```

Comment:
Can we have a loop here based on file name ?
["pom.xml", "./openbas-api/pom.xml", ...] for each
open
write